frontend/packages/console-app/src/components/quick-starts/data/serverless-application-quickstart.ts

```diff
@@ -1,3 +1,4 @@
+import { FLAG_KNATIVE_SERVING, FLAG_KNATIVE_EVENTING } from '@console/knative-plugin';
 import { serverlessIcon } from './tour-icons';
 
 export const serverlessApplicationQuickStart = {
```

Comment:
Add `: QuickStart` here as to ensure that the format and type definition matches.

```suggestion
export const serverlessApplicationQuickStart: QuickStart = {
```

Reply:
Done